examples/simple_look_transform.rs

```diff
@@ -39,10 +39,7 @@ fn setup(
 
     commands
         .spawn_bundle(LookTransformBundle {
-            transform: LookTransform {
-                eye: Vec3::new(-2.0, 2.5, 5.0),
-                target: Vec3::new(0.0, 0.5, 0.0),
```

Comment:
I actually prefer the old way because it gives names to the values.

Reply:
Done & done! I had troubles at first when I tried to put the enabled flag in the smoother, hence why I pushed the version in LookTransform. But actually, after revisiting it a few weeks later, I found a way to fix the issue: just need to reset the smoother state (self.lerp_tfm) when re-enabling it to make sure we don't jump from the last known transform to the current one!